tests/unit/lms/views/test_reports.py

```diff
@@ -29,7 +29,11 @@ def build_ai_from_pair(pair):
                 pair[0], pair[1], None, None, None
             )
 
-        app_instances = list(map(build_ai_from_pair, zip(test_urls, test_emails)))
+        app_instances = list(
+            map(  # pylint:disable=bad-builtin
+                build_ai_from_pair, zip(test_urls, test_emails)
+            )
+        )
```

Comment:
Is this different from?

```[build_ai_from_pair(pair) for pair in zip(test_urls, test_emails)]```

Reply:
I don't think so, I've replaced it with that